src/components/BShuttle/BShuttle.vue

```diff
@@ -0,0 +1,490 @@
+<template>
+  <div class='shuttle'>
+
+    <div class='b-shuttle-available'>
+      <label :id="`available-label-${id}`">{{ availableLabel }}</label>
+      <ul
+        :aria-labeledby="`available-label-${id}`"
+        role='selectbox'
+        tabindex='0'
+        class='input'
+      >
+        <li
+          v-if="placeholder && (!options || options.length === 0)"
+          :aria-label="placeholder"
+          class='placeholder'
+        >
+        {{ placeholder }}
+        </li>
+        <li
+          v-for="option in availableOptions"
+          :key="`available-option-${id}-${option.id}`"
+          :id="`available-option-${id}-${option.id}`"
+          :class="optionsClass(option, 'available')"
+          @click="onOptionClick(option, 'available', 'chosen', $event)"
+          @keyup.space.prevent.stop="onOptionClick(option, 'available', 'chosen', $event)"
+          @keyup.down.prevent.stop="onKeyupDown($event, option, 'available', 'chosen')"
+          @keyup.up.prevent.stop="onKeyupUp($event, option, 'available', 'chosen')"
+          @keyup.right.prevent.stop="onMoveSelectedClick(availableOptions,'available')"
+          @keyup.enter.prevent.stop="onMoveSelectedClick(availableOptions,'available')"
+          @keydown.up.prevent.stop
+          @keydown.down.prevent.stop
+          @keydown.space.prevent.stop
+          role='option'
+          tabindex='0'
+        >
+        {{ option.displayText }}
+        </li>
+      </ul>
+    </div>
+
+    <div class='b-shuttle-buttons'>
+      <button 
+        @click="onMoveAllOptions('available')"
+        aria-label="Move All To Chosen"
```

Comment:
All these buttons should have a title attribute instead of aria-label so that the title shows up on hover

Reply:
👍